src/controller/task.ts

```diff
@@ -0,0 +1,57 @@
+import { Request, Response, NextFunction } from 'express'
+import TaskModel, { Task } from '../model/task'
+
+export default {
+  async createTask (req: Request, res: Response, next: NextFunction): Promise<void> {
+    try {
+      const { title, description, importance } = req.body
+      const newTask: Task = await TaskModel.create({
+        title,
+        description,
+        importance
+      })
+      res.status(201).json(newTask)
+    } catch (err) {
+      next(err)
+    }
+  },
```

Comment:
Minor style improvement: there should be space (enter) between each method.

Reply:
fixed